scripts/reference.js

```diff
@@ -260,6 +271,12 @@ Game.prototype.reference = {
         'type': 'method',
         'description': 'Sets the background color of the given square.'
     },
+    'map.timeout': {
```

Comment:
This function was never documented for some reason, so, as part of the rewrite of the credits level and documenting new functions above, I decided to document it.